website/sidebars.js

```diff
@@ -151,6 +141,7 @@ export default {
     {
       type: "category",
       label: "Common Scripts",
+      label: "Script Examples",
```

Comment:
here are the most important production-level / well-known / wide-usage scripts in the ecosystem, so the name Script Examples is not suitable.